doc-content/shared-kie-docs/src/main/asciidoc/KieServer/Install-section.adoc

```diff
@@ -84,197 +85,377 @@ container. The Web container version of the WAR contains only the REST interface
 
+.System properties required for using a {CONTROLLER}
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`org.kie.server.id`
+|String
+|N/A
+|An arbitrary ID to be assigned to the server. If a {HEADLESS_CONTROLLER} is configured outside of {CENTRAL}, this is the ID under which the server will connect to the {HEADLESS_CONTROLLER} to fetch the KIE container configurations. If not provided, the ID is automatically generated.
+
+|`org.kie.server.user`
+|String
+|`kieserver`
+|The user name used to connect with the {KIE_SERVER} from the {CONTROLLER}, required when running in managed mode. Set this property in {CENTRAL} system properties. Set this property when using a {CONTROLLER}.
+
+|`org.kie.server.pwd`
+|String
+|`kieserver1!`
+|The password used to connect with the {KIE_SERVER} from the {CONTROLLER}, required when running in managed mode. Set this property in {CENTRAL} system properties. Set this property when using a {CONTROLLER}.
+
+|`org.kie.server.token`
+|String
+|N/A
+|A property that enables you to use token-based authentication between the {CONTROLLER} and the {KIE_SERVER} instead of the basic user name/password authentication. The {CONTROLLER} sends the token as a parameter in the request header. The server requires long-lived access tokens as the tokens are not refreshed.
+
+|`org.kie.server.location`
+|URL
+|N/A
+|The URL of the {KIE_SERVER} instance used by the {CONTROLLER} to call back on this server, for example: `http://localhost:8230/kie-server/services/rest/server`. Setting this property is required when using a {CONTROLLER}.
+
+|`org.kie.server.controller`
+|Comma-separated list
+|N/A
+|A comma-separated list of URLs to the {CONTROLLER} REST endpoints, for example `http://localhost:8080/{URL_COMPONENT_CENTRAL}/rest/controller`. Setting this property is required when using a {CONTROLLER}.
+
+|`org.kie.server.controller.user`
+|String
+|`kieserver`
+|The user name to connect to the {CONTROLLER} REST API. Setting this property is required when using a {CONTROLLER}.
+
+|`org.kie.server.controller.pwd`
+|String
+|`kieserver1!`
+|The password to connect to the {CONTROLLER} REST API. Setting this property is required when using a {CONTROLLER}.
+
+|`org.kie.server.controller.token`
+|String
+|N/A
+|A property that enables you to use token-based authentication between the {KIE_SERVER} and the {CONTROLLER} instead of the basic user name/password authentication. The server sends the token as a parameter in the request header. Note that long-lived access tokens are required as the tokens are not refreshed.
+
+|`org.kie.server.controller.connect`
+|Long
+|`10000`
+|The waiting time in milliseconds between repeated attempts to connect the {KIE_SERVER} to the {CONTROLLER} when the server starts.
+|===
 
-.Kie Server bootstrap switches
-[cols="1,1,1,1", frame="all", options="header"]
+ifdef::PAM,JBPM[]
+.System properties for persistence properties
```

Comment:
System properties for persistence *properties* doesn't make much sense. I would omit the *properties* word.

Reply:
Noted.